posthog/hogql_queries/actors_query_runner.py

```diff
@@ -123,6 +123,10 @@ def source_id_column(self, source_query: ast.SelectQuery | ast.SelectUnionQuery)
         else:
             select = source_query.select_queries[0].select
 
+        for column in select:
+            if isinstance(column, ast.Alias) and (column.alias in ("group_key", "actor_id", "person_id")):
```

Comment:
Why not move this into the loop right below 🙈
It's basically the condition that is there just modified.

Reply:
Mostly for precedence. Otherwise "session_id" or something may end up first and cause mayhem